evidence/athei/0001-retain-III.md

```diff
@@ -0,0 +1,67 @@
+# Evidence-0001: Retention at Rank III
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Report Date** | Date of submission (2024/08/26)                                                             |
+| **Submitted by**| Alexander Theißen                                                                           |
+
+
+## Member details
+
+- Matrix username: @alex:parity.io
+- Polkadot address: 15db5ksZgmhWE9U8MDq4wLKUdFivLVBybztWV8nmaJvv3NU1
+- Current rank: III
+- Date of initial induction: 2024/03/07
```

Comment:
Date of initial induction should read: "Seeding", in reference to: https://github.com/polkadot-fellows/seeding/pull/17.

Reply:
Are you sure? I thought this refers to the induction into the core fellowship. Not into the collective.

Reply:
Yes, this is about induction to the collective, though it is mostly relevant for promotion requests.